src/CommunityToolkit/Xamarin.CommunityToolkit/Views/DrawingView/Renderer/DrawingViewRenderer.tizen.cs

```diff
@@ -36,17 +36,17 @@ protected override void OnElementChanged(ElementChangedEventArgs<DrawingView> e)
 				SetNativeControl(canvasView);
 			}
 
-			if (e.OldElement != null)
+			if (e.OldElement != null && canvasView is not null)
```

Comment:
I'm definitely a fan of removing the `!` operator! One question though, is `canvasView` being null a valid scenario? If not should we still throw an exception so the consumer can detect something is wrong?

This potentially ripples through most of the changes but I didn't want to add loads of points if it is valid.

Reply:
In this case I don’t think it’s a good idea to throw the exception. Here we are cleaning all resources and unsubscribe from events (most likely user closed the page with control). Throwing exception here means user cannot navigate to another page.
Furthermore it is unlikely that canvasView will be null at this moment

Reply:
Ah yes that makes sense. I picked the worst example but just wanted to check we accept a `null` `canvasView` is a valid scenario in our renderer which it sounds like it is.

Reply:
actually no. the canvasView should always be initialized because it is actually the place where you draw. But because we initialize it here (not in ctor), we mark it as nullable and need to check that it is not null in a whole class.